checker/jdk/Makefile.jdk

```diff
@@ -30,8 +30,8 @@ build: classes
 
 ############## Validate annotated jdk against missing methods ##############
 
-SOURCES = $(shell find src/java src/javax -name '*.java')
-# CLASSES = $(shell find src/java src/javax -name '*.class')
+SOURCES = $(shell find src/java src/javax src/org/w3c src/org/xml -name '*.java')
```

Comment:
Good to know. What about the `com.sun.*` packages? These seem to be part of the `javac` API as opposed to the JDK itself.